src/plugins/data_explorer/public/components/app_container.tsx

```diff
@@ -39,6 +40,11 @@ export const AppContainer = React.memo(
       return <NoView />;
     }
 
+    trackUiMetric(DISCOVER_LOAD_EVENT);
+    if (isEnhancementsEnabled) {
+      trackUiMetric(NEW_DISCOVER_LOAD_EVENT);
```

Comment:
nit: did we want to mimic the `opt_in` `opt_out`  metrics. 

correct me if im wrong though, but i think this will just increment a counter any time it loads and passes this condition but it might hide when a user decides to initially enables it and decides they don't like the new discover so then they disable it. one user might not realize that there is a new Discover experience until a few weeks later so then the Discover load count would be way higher than the new Discover load count but they never opted out since they prefer the new Discover experience. But to us we might deduce that people do not like the new experience.

So the opt in and opt out count actually helps give us a better story that people opted in and decided to opt out

Reply:
Thanks @kavilla added a new metrics to track opt_in and opt_out